### PR TITLE
Resolved the NioDatagramChannel class reporting red problem

### PR DESCRIPTION
**Pull request by pengzhengfa**
Motivation:

Explain here the context, and why you're making that change.
What is the problem you're trying to solve.

Modification:

Describe the modifications you've done.

Result:

Fixes #<GitHub issue number>. 

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

**Comment by pengzhengfa** [comment]
In development, I often encounter a series of fatal problems in the production environment caused by naming problems. In high concurrency scenarios, serious problems are caused by some mistakes. We should reduce serious bugs in the program caused by naming problems

**Comment by amizurov** [comment]
Hmmm, strange to me could you please clarify your PR and squash commits.

**Comment by pengzhengfa** [comment]
@amizurov Don't you think dobind0, register0 method names are a little strange, and they don't conform to the style of seeing the name and knowing the meaning. In addition, in the development of nonstandard naming, there will be some unpredictable factors when it is high and distributed

**Comment by pengzhengfa** [comment]
@normanmaurer Hello, please check

File: transport/src/main/java/io/netty/channel/nio/NioEventLoop.java [hunk 1]
```diff
@@ -52,6 +52,7 @@
 /**
  * {@link SingleThreadEventLoop} implementation which register the {@link Channel}'s to a
  * {@link Selector} and so does the multi-plexing of these in the event loop.
+ * @author pengzhengfa
```
Comment (johnou) [inline transport/src/main/java/io/netty/channel/nio/NioEventLoop.java:55]: nah bro

File: transport/pom.xml [hunk 2]
```diff
@@ -17,7 +17,19 @@
...
   <modelVersion>4.0.0</modelVersion>
-  <parent>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
```
Comment (johnou) [inline transport/pom.xml:27]: nope, 1.6 for 4.1 branch.

**Comment by johnou** [comment]
> @amizurov Don't you think dobind0, register0 method names are a little strange, and they don't conform to the style of seeing the name and knowing the meaning. In addition, in the development of nonstandard naming, there will be some unpredictable factors when it is high and distributed

no, I don't find them strange, in fact the same pattern is used in the Java source. they are named like that because equivalent functions with same names exist in the code, the 0 suffix is just to distinguish it between native helper functions and public functions.

**Comment by johnou** [comment]
> @amizurov there will be some unpredictable factors when it is high and distributed

absolutely not.

**Comment by normanmaurer** [comment]
I agree with @johnou ... its just a matter of style and consistent within the code-base

**Pull request closed without merging**